pkg/rpcmiddleware/validator/validator_test.go

```diff
@@ -41,7 +41,7 @@ type msgWithValidate struct {
 	testSubject
 }
 
-func (m *msgWithValidate) Validate() error {
+func (m *msgWithValidate) ValidateFields() error {
```

Comment:
This is not the same as we have in our generated code

Reply:
Why did these not get caught by unit tests?